test/metabase/driver/postgres_test.clj

```diff
@@ -704,7 +704,7 @@
   (t2.with-temp/with-temp [Database database {:engine :postgres, :details (enums-test-db-details)}]
     (sync-metadata/sync-db-metadata! database)
     (f database)
-    (#'sql-jdbc.conn/set-pool! (u/id database) nil nil)))
```

Comment:
This line is the actual implementation of `invalidate-pool-for-db!` called now in `:sql-jdbc` implementation of method on following line.